api/src/main/resources/workbench-api.yaml

```diff
@@ -2415,14 +2415,14 @@ paths:
           description: The Terra Job for the extraction workflow
           schema:
             "$ref": "#/definitions/WgsCohortExtractionJobListResponse"
-  "/v1/workspaces/{workspaceNamespace}/{workspaceId}/cohorts/{cohortId}/extractGenomes":
+  "/v1/workspaces/{workspaceNamespace}/{workspaceId}/datasets/{datasetId}/extractGenomes":
     parameters:
       - "$ref": "#/parameters/workspaceNamespace"
       - "$ref": "#/parameters/workspaceId"
-      - "$ref": "#/parameters/cohortId"
+      - "$ref": "#/parameters/dataSetId"
     post:
       tags:
-        - cohorts
+        - dataSet
       description: 'Initiates a Terra Cromwell job to extract genomes of the given cohort.
```

Comment:
"of the given cohort" -> "of participants within the dataset"

or something similar

Reply:
argh, just now realizing how many places I used "Cohort" within our class names. I was thinking "Cohort" in terms of a group of participants and not as a concept within the workbench which is why I had used it so pervasively. I don't think we need to address all of them now if we want to get this out but maybe file a ticket to update those.

Maybe we can just generalize from `WgsCohortExtraction` -> `GenomeExtraction` or `WgsExtraction` wherever that is being used.

Reply:
Updated most of these in the PR. Remaining references:

- database / DAO (probably OK)
- WDL (also probably OK, "cohort extract" is the name of the workflow from variant store's perspective)
- WorkbenchConfig - possibly should be updated, not something I want to include in this PR though